strings/phobia.json

```
@@ -298,5 +298,16 @@
 		"glockroach",
 		"bee",
 		"sting"
+	],
+"ocky icky": [
+		"admin",
+		"admins",
+		"antagonist",
+		"antag",
+		"ban",
+		"banned",
+		"reddit",
+		"round",
+		"selfantag"
```

```suggestion
		"admin",
		"admins",
		"antagonist",
		"antag",
		"ban",
		"banned",
		"ick ock",
		"k",
		"leddit",
		"ocky",
		"ooc",
		"pwn",
		"reddit",
		"round",
		"selfantag",
		"self antag",
		"u",
		"uwa",
		"y",
```
words like lmao, stfu, lol, reddit, ahelp, pls, plz, etc. are already blocked by the config filter or whatever

---

The idea of searching the entire codebase for the word “reddit” and the only result will be brain trauma sounds funny to me

---

you can't even say the word reddit on tg servers

---

it could get removed from the filter at some point, though

---

ath made a policy feedback thread to remove reddit from the filter so he could say space reddit and i laughed when that happened

i'm not saying to remove it i just think it's funny